pycbc/events/stat.py

```diff
@@ -616,26 +614,35 @@ def rank_stat_single(self, single_info):
         numpy.ndarray
             The array of single detector statistics
         """
-        err_msg = "Sorry! No-one has implemented this method yet! "
-        raise NotImplementedError(err_msg)
+        return self.single(single_info[1])
 
-    def rank_stat_coinc(self, s, slide, step, to_shift,
-                        **kwargs): # pylint:disable=unused-argument
+    def rank_stat_coinc(self, sngls_list, slide, step, to_shift,
+                        **kwargs):  # pylint:disable=unused-argument
         """
         Calculate the coincident detection statistic.
         """
-        err_msg = "Sorry! No-one has implemented this method yet! "
-        raise NotImplementedError(err_msg)
+        rstat = sum(s[1]['snglstat'] ** 2 for s in sngls_list)
```

Comment:
I think a bit more documentation here would be good, the reference will be this 
https://iopscience.iop.org/article/10.3847/1538-4357/aa8f50#apjaa8f50eqn2 
ie eq.2 of the O2-statistic paper.

Reply:
Good point, will address.